specification/snippets/CorrelationHeader.yaml

```diff
@@ -1,4 +1,4 @@
 schema:
   type: string
   pattern: 11C46F5F-CDEF-4865-94B2-0EE0EDCC26DA
-description: 'The X-Correlation-ID from the request header, if supplied, mirrored back.'
\ No newline at end of file
+description: 'If the X-Correlation-ID header was included in the request the value of this header will be equal to that. Otherwise it will be a randomly chosen UUID.'
```

Comment:
`messageid` is not in the format of a UUID unless I am mistaken.

Reply:
true, its the rrt- sformat isnt it